deploy/playbooks/roles/common/templates/prod_nodes.py.j2

```diff
@@ -146,6 +146,16 @@ nodes = {
         'labels': ['ceph_build_xenial', 'amd64', 'x86_64', 'xfs', 'huge', 'rebootable', 'python3'],
         'provider': 'openstack'
     },
+    'leap15_huge': {
+        'script': dedent("""#!/bin/bash
+        zypper -n install python python-xml
```

Comment:
I would add `zypper -n ref` before this line, because sometimes repos get outdated.

Reply:
Thanks for the tip.  Done.